R/timeseriessummary-data.R

```diff
@@ -159,6 +160,7 @@ parseTSSRatingCurves <- function(reportData, timezone){
     curves[['applicablePeriods']] <- lapply(curves[['applicablePeriods']], function(p){
       p[['startTime']] <- formatOpenDateLabel(flexibleTimeParse(p[['startTime']], timezone))
       p[['endTime']] <- formatOpenDateLabel(flexibleTimeParse(p[['endTime']], timezone))
+      p <- p[order(p[['startTime']]),]
```

Comment:
yeah I wasn't really sure what made sense and if I should sort before getting into the lapply. I can try mocking up some data to verify and try sorting ahead of time. Thanks!